Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/shotgun.yml

```diff
@@ -10,7 +10,8 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 8  # Should have the same or less damage than a regular pellet
+        Piercing: 8
+        Blunt: 20
```

Comment:
why blunt

Reply:
I think it gives it a neat reason to exist since it'll still hit like a truck if you have on piercing-heavy armor which most is.

Reply:
> because it's a large chunk of metal that'll impact you and fall to the ground rather than a small one that'll break into your skin, just makes more sense to have it do blunt damage like beanbags. If you want me to adjust the ratio I can.

That's real life logic but I'm wondering from a game perspective why it doesn't align with other projectiles.

Reply:
You do realize that by making it blunt it bypasses armor vests right? If your goal is to do that, maybe consider using the actual armor piercing modifier? You can find it on any AP ammo

Reply:
Giving 1 specific bullet type armor piercing via damage types doesn't seem like a great idea.